images.css

```diff
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
```

Comment:
Use Prettier to format your code. This will improve readability of your code.

Reply:
Yes Im using